src/truncate.js

```diff
@@ -4,38 +4,42 @@ const AlpineTruncateMagicMethod = {
     start() {
         checkForAlpine()
 
-        Alpine.addMagicProperty('truncate', function () {
+        Alpine.addMagicProperty('truncate', () => {
```

Comment:
This change technically doesn't belong to this PR.
Using an arrow function or a classic function expression, since we don't reference `this` is pretty much the same here.

I would prefer we keep the changes into the scope of this PR, would you mind reverting it, please?

P.s. I'm not against the use of the arrow function here but, for consistency, we should update it on all the other helpers, too.
Feel free to open a different PR where you update all the existing helpers.

Reply:
It's required to access the `addEllipsis` method, right?

Reply:
I also used this pattern in the `$undo` helper as well. I'll create a PR to update the whole codebase now.

Reply:
yeah, sorry, I shouldn't review PR later at night. I didn't notice that you added the function in the scope of AlpineTruncateMagicMethod 🤦  rather than a helper.